sql/item_latents.sql

```diff
@@ -3234,4 +3234,57 @@ INSERT INTO `item_latents` VALUES(21661, 405, 3, 10, 0);      -- Rune Algol: Dra
 -- INSERT INTO `item_latents` VALUES(21667, 25, 10, ??, 0); -- Futhark Claymore: (D): Accuracy+10
 -- INSERT INTO `item_latents` VALUES(21667, 30, 10, ??, 0); -- Futhark Claymore: (D): Magic Accuracy+10
 -- INSERT INTO `item_latents` VALUES(21668, 25, 10, ??, 0); -- Peord Claymore: (D): Accuracy+10
--- INSERT INTO `item_latents` VALUES(21668, 30, 10, ??, 0); -- Peord Claymore: (D): Magic Accuracy+10
\ No newline at end of file
+-- INSERT INTO `item_latents` VALUES(21668, 30, 10, ??, 0); -- Peord Claymore: (D): Magic Accuracy+10
+
+-- -------------------------------------------------------
+-- Conqueror Enhances Berserk I - V
+-- -------------------------------------------------------
+INSERT INTO `item_latents` VALUES(18991,62,5,13,56);   -- Conqueror 75 Attack +5% if Berserk Active
```

Comment:
BG Wiki states the following (For the lvl 75 version):

> Augments Berserk effect gives the following:
> Attack to +30% (±77/256), Defense -30% (Upgraded from the original Berserk effect of +25%/-25%)
> Effect on Attack persists even after Conqueror removal.
> Critical Hit Rate +5%
> Effect on Critical Hit Rate does not persist when Conqueror is removed.
> May also give some amount (3~5%) of Double Attack.

Your value says +5%

Reference: https://www.bg-wiki.com/bg/Conqueror_(Level_75)

You may need to recheck your numbers.